docs/useListController.md

```diff
@@ -54,6 +55,7 @@ import {
 } from 'react-admin';
 
 const MyList = ({
+    customStoreKey = '',
```

Comment:
don't add that to the general example, it brings confusion.

Reply:
Aren't they the default values?